### ISSUE TITLE
Firewall issues - evilginx is only listening on IPv6

### ISSUE THREAD

**Issue by FranklinClinton** [kgretzky/evilginx2#229]
Hi there,

testing a little bit, but got into trouble. When I try to reach the server via webbrowser I get a timeout. I have allowed TCP 80 and TCP 443 in iptables. I've got a lot of iptables so I can assume that the problem is not there.
`netstat -tulpn`
`tcp6       0      0 :::80                   :::*                    LISTEN      708/evilginx`
`tcp6       0      0 :::443                  :::*                    LISTEN      708/evilginx`
`udp6       0      0 :::53                   :::*                                708/evilginx`

Seems like evilginx is not listening on IPv4. I disabled IPv6 on my ubuntu server vps successfully. Any ideas how I can get evilginx to listen on ipv4?

**Comment by FranklinClinton** [comment]
Never mind - fixxed it

**Comment by Bacalau** [comment]
How did you fix this issue?

**Comment by pawp81** [comment]
@FranklinClinton I have same question. How did you fix it? I edited http_proxy.go and replaced net.Listen("tcp" with net.Listen("tcp4". Then I recompiled go source code. After starting evilginx port 443 uses Ipv4, but port 80 uses Ipv6
tcp        0      0 0.0.0.0:22              0.0.0.0:*               LISTEN      935/sshd            
tcp        0      0 0.0.0.0:443             0.0.0.0:*               LISTEN      4610/evilginx       
tcp6       0      0 :::80                   :::*                    LISTEN      4610/evilginx       
tcp6       0      0 :::22                   :::*                    LISTEN      935/sshd            
udp6       0      0 :::53                   :::*                                4610/evilginx
